client/src/views/admin/AdminRegistration.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div>
+      <b-container>
+        <b-card>
+          <b-container>
+            <h1>
+              <b>ADMIN REGISTRATION </b>
+            </h1>
+            <div class="form mb-3">
+              <b-form-input
+                v-model="name"
+                :state="userState"
+                placeholder="name"
+              ></b-form-input>
+            </div>
+            <div class="form mb-3">
+                <b-form-input
+                  v-model="username"
+                  :state="userState"
+                  placeholder="username"
+                ></b-form-input>
```

Comment:
as we discussed yesterday we are discarding `username` for admin, so you discard here as well